compiler/tflchef/proto/tflchef.proto

```diff
@@ -333,6 +333,10 @@ message PowOptions {
   // None
 }
 
+message PReluOptions {
```

Comment:
I think we don't need this. In exporter code as you wrote there was no PRelu options builder.
```cxx
  // Make PRelu operator; PRelu does not have Options
  auto op_offset = CreateOperator(builder, op_idx, inputs, outputs);
```